connector/connect/common/src/main/protobuf/spark/connect/relations.proto

```diff
@@ -570,3 +571,21 @@ message Hint {
   // (Optional) Hint parameters.
   repeated Expression.Literal parameters = 3;
 }
+
+// Unpivot a DataFrame from wide format to long format, optionally leaving identifier columns set.
+message Melt {
+  // (Required) The input relation.
+  Relation input = 1;
+
+  // (Required) Id columns.
+  repeated Expression ids = 2;
```

Comment:
The ids in Dataset.melt is `ids: Array[Column]`, I'm not sure it's appropriate.
@amaliujia @zhengruifeng @HyukjinKwon

Reply:
this is correct! we use expression instead of columns